src/main/java/me/confor/velocity/chat/modules/GlobalChat.java

```diff
@@ -66,57 +64,61 @@ public void onPlayerChat(PlayerChatEvent event) {
             msg = msg.replaceText(config.urlReplacement);
 
         if (config.GLOBAL_CHAT_PASSTHROUGH)
-            sendMessage(msg,currentServer.get().getServer());
-        else
             sendMessage(msg);
+        else
+            sendMessage(msg, currentServer.get().getServer());
 
         if (config.GLOBAL_CHAT_TO_CONSOLE)
             this.logger.info("GLOBAL: <{}> {}", player, message);
 
-        if (!config.GLOBAL_CHAT_PASSTHROUGH)
+        /*
+        broken >=1.19.1
+
+        if (config.GLOBAL_CHAT_PASSTHROUGH)
             event.setResult(ChatResult.denied());
+         */
```

Comment:
> because using other plugins or patches to Velocity you can get it working

UnsignedVelocity can do that: https://modrinth.com/plugin/unsignedvelocity

It's probably at least worth mentioning that such a plugin would be required to use `GLOBAL_CHAT_PASSTHROUGH` on >=1.19.1, maybe in the config file or README.md.